.github/workflows/new-line.yml

```diff
@@ -0,0 +1,14 @@
+name: Code Hygiene
```

Comment:
I'd recommend renaming the file name to match the workflow name, `new-line` -> `code-hygiene`.  In the future you could add addition jobs that scanned for other hygiene related issues.